src/components/textField/TextField.driver.new.ts

```diff
@@ -0,0 +1,94 @@
+import _ from 'lodash';
+import {fireEvent} from '@testing-library/react-native';
+import {TextFieldProps} from './types';
+import {useComponentDriver, ComponentProps} from '../../testkit/new/Component.driver';
```

Comment:
Is this a good name for the driver arguments? Perhaps ComponentDriverArguments or DriverArguments will be better?

Reply:
If you're talking about `ComponentProps` then this is not the PR to change it; feel free to create a PR for it.